documentation/modules/configuring/proc-generating-reassignment-json-files.adoc

```diff
@@ -64,7 +64,7 @@ metadata:
     strimzi.io/cluster: my-cluster
 spec:
   partitions: 10
-  replicas: 3
+  replicas: 4
```

Comment:
Why exactly do we need 4 here? It is a bit more unusual replication factor then 3 and there does not seem to be any relation to the rest of the procedure.

Reply:
Yeah. I made a few changes to this procedure as it's a pre-req to the reassign and topic replication procedures. I was trying to make it more easily understood for anyone wanting to understand the example we use in the topic replication procedure. But I don't think we need it and, as you say, it's not a great example here. Reverted.